docs/ref/django-admin.txt

```diff
@@ -1467,10 +1467,12 @@ Enables :ref:`SQL logging <django-db-logger>` for failing tests. If
 Runs tests in separate parallel processes. Since modern processors have
 multiple cores, this allows running tests significantly faster.
 
-By default ``--parallel`` runs one process per core according to
-:func:`multiprocessing.cpu_count()`. You can adjust the number of processes
-either by providing it as the option's value, e.g. ``--parallel=4``, or by
-setting the :envvar:`DJANGO_TEST_PROCESSES` environment variable.
+Using ``--parallel`` without a value, or with the value ``auto``, runs one test
+process per core according to :func:`multiprocessing.cpu_count()`. You can
+override this by passing the desired number of processes, e.g.
+``--parallel 4``. You can also use ``--parallel`` without a value and set the
+:envvar:`DJANGO_TEST_PROCESSES` environment variable to the desired number of
+processes.
```

Comment:
It is a change in documented behavior. Previously it was enough to set `DJANGO_TEST_PROCESSES` without passing `--parallel`, e.g.
-  before
   ```
   $ export DJANGO_TEST_PROCESSES=3
   $ ./runtests.py queries
   Testing against Django installed in '/django/django' with up to 3 processes
   Found 416 test(s).
   Creating test database for alias 'default'...
   Cloning test database for alias 'default'...
   Cloning test database for alias 'default'...
   Cloning test database for alias 'default'...
   System check identified no issues (1 silenced).
   .......ssss.sss..s..................s............................................s...............s..........x..s.....................x................................................................................................................s.........................................................................................................................................................................
   ----------------------------------------------------------------------
   Ran 416 tests in 0.575s

   OK (skipped=13, expected failures=2)
   Destroying test database for alias 'default'...
   Destroying test database for alias 'default'...
   Destroying test database for alias 'default'...
   Destroying test database for alias 'default'...
   ```
- with this patch:
  ```
   $ export DJANGO_TEST_PROCESSES=3
   $ ./runtests.py queries
   Testing against Django installed in '/home/felixx/repo/django/django'
   Found 416 test(s).
   Creating test database for alias 'default'...
   System check identified no issues (1 silenced).
   .................................ssss.sss..s.............................s...............s........s...........................x..........s.............x.........................................................................................................................................................................s..............................................................................................
   ----------------------------------------------------------------------
   Ran 416 tests in 1.059s

   OK (skipped=13, expected failures=2)
   Destroying test database for alias 'default'...
   ```

Reply:
Good point, thanks for picking that up. I've rearranged the parsing so this is not the case.